src/repositories/admins-repository.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
+const Sequelize = require('sequelize');
+
+const { Op } = Sequelize;
+
+const { Admin } = models;
+
 module.exports = function createAdminsRepository(models) {
-  const { Admin } = models;
+  function list(queryParams) {
+    const { limit, offset, contains } = queryParams;
+
+    const getData = Admin.findAll({
+      offset,
+      limit,
+      where: { firstName: { [Op.like]: [`%${contains}%`] } },
+    });
+
+    const getTotal = Admin.findAndCountAll();

```

Comment:
http://docs.sequelizejs.com/manual/tutorial/models-usage.html#-findandcountall-search-for-multiple-elements-in-the-database-returns-both-data-and-total-count

you do not need the first request, `findAndCountAll` return rows and total :)

Reply:
first request is for selected result with limits and offset from database, and the second is for counting all the rows in database, despite the selection..